dexlib2/src/main/java/org/jf/dexlib2/dexbacked/DexBackedDexFile.java

```diff
@@ -138,6 +182,10 @@ public DexBackedDexFile(@Nullable Opcodes opcodes, @Nonnull byte[] buf) {
         this(opcodes, buf, 0, true);
     }
 
+    public DexBackedDexFile(@Nullable Opcodes opcodes, @Nonnull MappedDexBuffer buf) {
```

Comment:
I don't think this constructor is needed now?

Reply:
Done. Thanks